UnitsNet/Scripts/UnitDefinitions/MassFlow.json

```diff
@@ -0,0 +1,31 @@
+{
+    "Name": "MassFlow",
+    "BaseUnit": "KilogramPerSecond",
+    "XmlDoc": "",
+    "Units": [
+        {
+            "SingularName": "KilogramPerSecond",
+            "PluralName": "KilogramsPerSecond",
+            "FromUnitToBaseFunc": "x",
+            "FromBaseToUnitFunc": "x",
+            "Localization": [
+                {
+                    "Culture": "en-US",
+                    "Abbreviations": ["kg/s"]
+                }
+            ]
+        },
+		{
+            "SingularName": "TonnePerDay",
+            "PluralName": "TonnesPerDay",
+            "FromUnitToBaseFunc": "x*86.4000",
+            "FromBaseToUnitFunc": "x/86.4000",
```

Comment:
These two functions should be swapped I think?

1 kg/s = 86.4 t/d
https://www.google.no/search?q=Tonnes+Per+Day+to+kilograms+per+second&oq=Tonnes+Per+Day+to+kilograms+per+second&aqs=chrome..69i57.6551j0j7&sourceid=chrome&es_sm=93&ie=UTF-8#q=kilograms+per+second+to+Tonnes+Per+Day+